### PR TITLE
Lint CSS files on precommit

### PR DESCRIPTION
**Pull request by AlexeyKorkoza**
I write the parameters for checking files of css in a .csslint file. I run the csslint-cli and it write the same mistakes , but the parameters are constantly doing different. Tell me what went wrong. How to write parameters correctly ?

I write in command line: node_modules/csslint-cli/bin/csslint-cli .csslint css/  .
node_modules/csslint-cli/bin/csslint-cli  - path to npm package
.csslint - a file with c written parameters.
css/ - a folder what checking a npm package

### CODE REVIEW

File: .csslint [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (irnc) [inline .csslint]: This file should be named `.csslintrc`, please read documentation at https://github.com/CSSLint/csslint/wiki/command-line-interface#configuration-files

File: package.json [hunk 2]
```diff
@@ -17,6 +18,8 @@
   },
   "homepage": "https://github.com/pub-t/plain-destination#readme",
   "devDependencies": {
+    "csslint": "^0.1.0",
+    "csslint-cli": "0.0.3",
```
Comment (irnc) [inline package.json]: Could we use standard CLI from `csslint` package or do we specifically need alternative `csslint-cli`? Documentation for standard CLI could be seen at https://github.com/CSSLint/csslint/wiki/command-line-interface#running-on-nodejs

File: package.json [hunk 3]
```diff
@@ -4,6 +4,7 @@
   "description": "<h1><b>Plain-destination</b></h1>\r <h2>Structure</h2>\r | filename  | Content a file |\r | ------------- | ------------- |\r | index.html  | Main file in page-proofs  |\r | css/style.css  | Main styles  |\r | css/reset.css  | Reset CSS of Eric Mayer |\r | css/fonts.css  | Connect different the fonts |\r | css/media.css  | Main styles  |\r | js/script.css  | Work for element of form |",
   "main": "index.js",
   "scripts": {
+    "csslint": "csslint .css",
```
Comment (irnc) [inline package.json]: Expected usage is `csslint [options] [file|dir]*`, i.e. our `csslint .css` would try to lint non-existing place named `.css`.

See https://github.com/CSSLint/csslint/wiki/command-line-interface#running-on-nodejs for details.

**Comment by KrivichaninDenis** [comment]
Что с этим pr?он в in progress?

**Comment by AlexeyKorkoza** [comment]
Я cейчас перенесу из backlog в in progress #19 . Я создал pr для #19, чтобы мне подсказали, где я сделал ошибки.

**Comment by AlexeyKorkoza** [comment]
I understood where they were made the errors.

File: package.json (hunk 2 above)
Comment (AlexeyKorkoza) [inline package.json]: I use standard CLI from csslint

**Comment by AlexeyKorkoza** [comment]
I close this pull request because commits go out of order in this branch.

**Pull request closed without merging**